### DISCUSSION TITLE
✈ Infinite Animations (set animation-iteration-count from the UI) (Migrated)

### DISCUSSION THREAD

**Discussion by copen8** [elementor/elementor#30214] [Feature Request]
Hello! Thank for the best page builder ever. :D 

I have a question about animations. I know that you have a great feature which make it possible to add animation when you hover over for example a icon. 
But it would be cool to be able to have the same animation even if you don't hover. 

For example an arrow that bounce all the time/infinite. See the links 
 https://www.squarespace.com/
https://codepen.io/dodozhang21/pen/siKtp

**Comment by arielk** [comment]
Infinity Animation could be an interesting idea.
Is it relevant to all elements or only for an icon or a button?

**Comment by copen8** [comment]
I don't know if it is relevant for all elements, I just wanted it for my icon and button. As showned in the example with the animation of the arrow that bounces. 

And I made a dummy page which I built with Elementor there I added some css-code to make my arrow bounce without hovering. See link below
(
http://joakim.joakimwennersten.se/)

**Comment by ciriousjoker** [comment]
Icons and buttons would be cool, but technically it shouldn't be that hard to just make it available for all elements

**Comment by 79ho3ein** [comment]
hello @arielk 
Just like the tooltip animation in the Hotspot widget. These animations will be great for the button widget.

**Comment by nicholaszein** [comment]
Hi there,

### ✈️ This Feature Request has been migrated from issue #684 

if you still think this request should be considered for our roadmap, please cast your **⬆️ Upvote** on the first comment.

Don't worry, your comments, reactions, and participation in the old issue will be accounted for when considering this Feature Request for our Roadmap.

Happy voting! 🙂
